test/onnx/test_export_modes.py

```diff
@@ -66,10 +61,10 @@ def foo(a):
             return (a, a)
         f = io.BytesIO()
         x = torch.ones(3)
-        torch.onnx._export(foo, (x,), f)
```

Comment:
in `torch.onnx._export`, it has a parameter called export_type that is not exposed in `torch.onnx.export`, but we have its tests right above blocks. I am thinking if we should delete those as it's not exposed anyway?

Reply:
That should be fine. We don't need to export to all formats. ONNX uses protobuf anyway